exercises/risk-risiko.cpp

```diff
@@ -27,3 +26,48 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+#include <iostream>
+#include <ctime>
+using namespace std;
+
+void swap(int& a, int& b) {
+    int temp = a;
+    a = b;
+    b = temp;
+}
+
+void selectionsort(int array[], int n) {
+    int indexMax;
+    for (int i = 0; i < n; i++) {
+        indexMax = i;
+        for (int j = i + 1; j < n; j++) {
+            if (array[j] > array[indexMax])
+                indexMax = j;
+        }
+        swap(array[i], array[indexMax]);
+    }
+}
+
+int main() {
+    srand(time(0));
+    int Red[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
+    int Blue[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
+    selectionsort(Red, 3);
+    selectionsort(Blue, 3);
+    cout << "Red dices: \n" << Red[0] << " (N) \n"
+        << Red[1] << " (M) \n"
+        << Red[2] << " (O) \n\n";
+
+    cout << "Blue dices: \n" << Blue[0] << " (N) \n"
+        << Blue[1] << " (M) \n"
+        << Blue[2] << " (O) \n\n";
+
+    for (int i = 0; i < 3; i++) {
```

Comment:
set 3 in a separate CONSTANT to be compliant to the SOLID principles and do not use magic number or magic string